completions/bash/poudriere

```diff
@@ -0,0 +1,437 @@
+                esac
+                ;;
+
+            ports)
+                case "$prev" in
+                    -B)
+                        # XXX: Maybe there are not that many branches and it is possible to actually list them here.
+                        compopt -o default
+                        ;;
+                    -f)
+                        _poudriere_filesystem
+                        ;;
+                    -m)
+                        COMPREPLY=($(compgen -W 'portsnap git null svn svn+http svn+https svn+file svn+ssh' -- "$cur"))
+                        ;;
+                    -M)
+                        compopt -o default
+                        ;;
+                    -p)
+                        _poudriere_ports_list
+                        ;;
+                    *)
+                        COMPREPLY=($(compgen -W "$(_parse_help poudriere $selected_command)" -- "$cur"))
+                        ;;
+                esac
+                ;;
+
+            pkgclean)
+                case "$prev" in
+                    -f)
+                        _poudriere_absolute_path
+                        ;;
+                    -j)
+                        _poudriere_jails_list
+                        ;;
+                    -J)
+                        _poudriere_jobs_number
+                        ;;
+                    -p)
+                        _poudriere_ports_list
+                        ;;
+                    -z)
+                        _poudriere_set_list
+                        ;;
+                    *)
+                        if [[ $cur != -* ]]; then
+                            _poudriere_direct_port
+                        fi
+                        if [[ ${COMPREPLY[@]} -eq 0 ]]; then
+                            COMPREPLY=($(compgen -W "$(_parse_help poudriere $selected_command)" -- "$cur"))
+                        fi
+                        ;;
+                esac
+                ;;
+
+            options)
+                case "$prev" in
+                    -a)
+                        _poudriere_architecture
+                        ;;
+                    -f)
+                        _poudriere_absolute_path
+                        ;;
+                    -j)
+                        _poudriere_jails_list
+                        ;;
+                    -p)
+                        _poudriere_ports_list
+                        ;;
+                    -z)
+                        _poudriere_set_list
+                        ;;
+                    *)
+                        if [[ $cur != -* ]]; then
+                            _poudriere_direct_port
+                        fi
+                        if [[ ${COMPREPLY[@]} -eq 0 ]]; then
+                            COMPREPLY=($(compgen -W "$(_parse_help poudriere $selected_command)" -- "$cur"))
+                        fi
+                        ;;
+                esac
+                ;;
+
+            # TODO: queue) ;;
```

Comment:
Don't bother as the command is unstable and may go away.

Reply:
Done.